GitHub workflow issue

Running packwiz failed 
https://github.com/Fabulously-Optimized/fabulously-optimized/blob/main/.github/workflows/auto-publish.yml#L49
https://github.com/Fabulously-Optimized/fabulously-optimized/actions/runs/9453274650/job/26038486203

> Run chmod +x ./packwiz
> /home/runner/work/_temp/71b77c9e-20e0-4928-b32d-a6e873d26d20.sh: line 2: pushd: Packwiz/1.20.6: No such file or directory

> Run cp  cosign-bundle.json
> cp: missing destination file operand after 'cosign-bundle.json'
> Try 'cp --help' for more information.
> Error: Process completed with exit code 1.

https://github.com/Fabulously-Optimized/fabulously-optimized/actions/runs/9470547995/job/26091726338

--- a/.github/workflows/auto-publish.yml
+++ b/.github/workflows/auto-publish.yml
@@ -59,7 +59,7 @@ jobs:
           subject-path: Fabulously.Optimized-${{ github.ref_name }}.mrpack
        - name: Copy and zip attestation
          run: |
-           cp ${{ steps.attest-mrpack.bundle-path }} cosign-bundle.json
+           cp ${{ steps.attest-mrpack.outputs.bundle-path }} cosign-bundle.json
            zip cosign-bundle.zip cosign-bundle.json
        - name: Publish to Modrinth
          uses: Kir-Antipov/mc-publish@v3.3

